javascript/uber_search.js

```diff
@@ -52,28 +57,56 @@ var UberSearch = function(data, options){
     view: {
       renderResults: renderResults,
       buildResult: options.buildResult || buildResult,
-      keypressInput: options.search ? searchField.input : outputContainer.view
+      keypressInput: options.search ? searchField.input : null
```

Comment:
Is this necessary anymore now that the results receive focus? Can we just always listen to the `resultsContainer`?

Reply:
Yes, it's still necessary.

Reply:
What happens when we change the keypress input to be the `outputContainer.view`?